GliaWidgets/Sources/Theme/Theme+SnackBarStyle.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+extension Theme {
+
+    var snackBarStyle: SnackBarStyle {
+        .init(
+            background: color.baseDark,
+            textColor: color.baseLight,
+            textFont: .font(weight: .regular, size: 17),
+            accessibility: .init(isFontScalingEnabled: true)
+        )
+    }
+
+    var invertedSnackBarStyle: SnackBarStyle {
+        .init(
+            background: color.baseLight,
+            textColor: color.baseDark,
+            textFont: .font(weight: .regular, size: 17),
+            accessibility: .init(isFontScalingEnabled: true)
+        )
+    }
+
+    public struct SnackBarStyle: Equatable {
+        /// View's background color
+        public var background: UIColor
+        /// Snack message text color.
+        public var textColor: UIColor
+        ///
+        public var textFont: UIFont
+
+        public var accessibility: Accessibility
+    }
+}
+
+extension Theme.SnackBarStyle {
+    public mutating func apply(
```

Comment:
I guess it should be internal

Reply:
fixed - 4222b481329e90ba6ba9cbff82dcbf7851515df1

Reply:
fixed - def62f8f39e255627ae825ed53e368ded82788d7